src/System.Private.CoreLib/shared/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -3107,8 +3107,8 @@ internal static Attribute GetCustomAttributeHelper(Type type, Type attributeType
 
                         foreach (CustomAttributeNamedArgument namedArgument in data.NamedArguments)
                         {
-                            PropertyInfo p = t.GetProperty(namedArgument.MemberInfo.Name, BindingFlags.Public | BindingFlags.Instance);
-                            object value = namedArgument.TypedValue.Value;
+                            PropertyInfo p = t.GetProperty(namedArgument.MemberInfo.Name, BindingFlags.Public | BindingFlags.Instance)!;
```

Comment:
Can you add `// TODO-NULLABLE: [https://github.com/dotnet/roslyn/issues/26761`](https://github.com/dotnet/roslyn/issues/26761`) on this?  I'd like to track the potential for some attribute that supports the `throwOnError:true` case.

Reply:
sure